mattermost/postgres.yaml

```diff
@@ -12,3 +12,6 @@ spec:
     image: postgres:latest
```

Comment:
We should pin it to a specific version, otherwise we'll run into issues again sooner or later.